server/config/routes.js

```diff
@@ -54,7 +54,7 @@ module.exports = function(app, passport) {
   // fetched and seed our stores with data.
   // App is a function that requires store data and url to initialize and return the React-rendered html string
   app.get('*', function (req, res, next) {
-    App(req, res);
+    App.default(req, res);
```

Comment:
@choonkending let wait till babel6 release the new version that we could do merge, then we'll discuss about this :smile: